### PR TITLE
bug: Ignore pre-release version

### LINKED ISSUE #17: Repos didn't update to v1.0.1
Repos using tfupdate 1.0.0 didn't get any PR for Terraform 1.0.1. 
However, repo still using tfupdate 0.1.0 opened PR for Terraform 1.0.1 (https://github.com/HENNGE/shobosso-unreadchecker/pull/303)

Last week a PR was opened in repos using tfupdate 1.0.0 to update to Terraform 1.1.0 Alpha. (https://github.com/HENNGE/shobo-support-viewer/pull/73) 
Could this be a bug?

### PR DESCRIPTION
**Pull request by sharu1204**
Resolves #17

### CODE REVIEW

**Review by michaelaw320 — APPROVED** [review]
looks legit regex 😄

**Comment by hanscg** [comment]
Sorry, I think the problem that I wrote on the issue happened because those repos used 1.0.0 instead of the latest 1.0.1.
Looking at https://github.com/minamijoyo/tfupdate/releases/tag/v0.5.0, it seems that it should have automatically ignored pre-releases.
I think we can revert to using `tfupdate release latest` instead of `list`?

**Comment by hanscg** [comment]
`tfupdate release list hashicorp/terraform` result:
image

`tfupdate release latest hashicorp/terraform` result:
image

**Comment by sharu1204** [comment]
@hanscg 
Thanks for the heads-up! I updated it.

File: entrypoint.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ set -e
 function run_tfupdate {
   case ${INPUT_RESOURCE} in
     terraform)
-      VERSION=$(tfupdate release list hashicorp/terraform| grep -v ".*-rc[0-9]$"| sort -r| head -1)
+      VERSION=$(tfupdate release latest hashicorp/terraform)
```
Comment (hanscg) [inline entrypoint.sh:12]: Use latest in line 22 and 31 as well?

Comment (sharu1204) [reply entrypoint.sh:12]: Fixed

**Review by hanscg — APPROVED** [review]
LGTM!
Thank you for dealing with this 👍

**Pull request merged**